### PR TITLE
PFC-28 pet api endpoints

### PR DESCRIPTION
**Pull request by blaizans**

### CODE REVIEW

File: Cloud/src/main/java/com/example/petfeedercloud/dtos/PetDTO.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
+package com.example.petfeedercloud.dtos;
+
+import java.util.Date;
+
+public class PetDTO {
+    private Long petId;
+    private Long userId;
+    private String name;
+    private Date birthdate;
+    private int weight;
```
Comment (FuLLeNN) [inline Cloud/src/main/java/com/example/petfeedercloud/dtos/PetDTO.java:10]: Should be a double or float

**Review by FuLLeNN — CHANGES_REQUESTED** [review]
Change the variable type and pr

File: Cloud/src/main/java/com/example/petfeedercloud/dtos/PetDTO.java (hunk 1 above)
Comment (blaizans) [reply Cloud/src/main/java/com/example/petfeedercloud/dtos/PetDTO.java]: Yeah on point

**Review by FuLLeNN — APPROVED** [review]

**Comment by FuLLeNN** [comment]
Change Pet.java weight too please.

**Comment by blaizans** [comment]
just did that hoe

**Comment by FuLLeNN** [comment]
> just did that hoe

Hell done colleague.

**Comment by FuLLeNN** [comment]
> > just did that hoe
> 
> Hell done colleague.

Well*.

**Pull request merged**